SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/FileHelper.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.AppCenter.Utils
+{
+    /// <summary>
+    /// Utilities for the file system.
+    /// </summary>
+    public class FileHelper
+    {
+        private readonly DirectoryInfo _baseDirectory;
+
+        /// <summary>
+        /// Creates an instance. This parameterless constructor is needed for tests.
+        /// </summary>
+        public FileHelper()
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance with the given base path.
+        /// </summary>
+        /// <param name="basePath">The path to use as the base for all operations.</param>
+        public FileHelper(string basePath)
+        {
+#if WINDOWS_UWP
+            var basePathPrefix = global::Windows.Storage.ApplicationData.Current.LocalFolder.Path;
+#else
+            //TODO what should this be?
```

Comment:
TODO

Reply:
Yep, known TODO - which is why we have "do not merge"

Reply:
This is to make sure its tracked as a requested change before merge, todo are often left out ;)